CMakeLists.txt

```diff
@@ -65,7 +65,14 @@ check_include_file ( stdio.h HAVE_STDIO_H )
 check_include_file ( math.h HAVE_MATH_H ) 
 check_include_file ( errno.h HAVE_ERRNO_H ) 
 check_include_file ( stdarg.h HAVE_STDARG_H ) 
-check_include_file ( unistd.h HAVE_UNISTD_H ) 
+check_include_file ( unistd.h HAVE_UNISTD_H )
+
+if ( WIN32 )
+  include ( CheckIncludeFiles )
```

Comment:
`include ( CheckIncludeFiles )` should be redundant here, as `check_include_files` isn't used.